### PR TITLE
[DOC] Add a section about the different container tags provided

### PR DESCRIPTION
**Pull request by Nexucis**
I'm adding a little description about the different tags we are provided. Maybe it can help a bit to understand which one you should use depending on the context.

I'm also taking the opportunity to cleanup and rewording a bit the rest of the documentations.

For example, I have removed the section `Note: What to do with failures?`. Someone that knows how to use docker/podman is already aware about how to destroy a container.

/cc @eschabell

### CODE REVIEW

File: docs/user-guides/installing-perses-in-a-container.md [hunk 1]
```diff
@@ -28,39 +27,36 @@ $ docker run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
...
+- precise release image: `persesdev/perses:v0.42.1`, `persesdev/perses:v0.42.1-debug`, `persesdev/perses:v0.42.1-distroless`, `persesdev/perses:v0.42.1-distroless-debug`
+- minor release image: `persesdev/perses:v0.42`, `persesdev/perses:v0.42-debug`, `persesdev/perses:v0.42-distroless`, `persesdev/perses:v0.42-distroless-debug`
 
 ## Connect a browser (default)
 
 Open the Perses console at http://localhost:8080.
 
-You are presented with an empty home dashboard, in light mode. For fun you can choose optionally to flip the switch in
-the top right corner to enable dark mode.
+You are presented with an empty home dashboard, in light mode.
```
Comment (Gladorme) [inline docs/user-guides/installing-perses-in-a-container.md:57]:
```suggestion
You are presented with the home page, in light mode.
```
no ?

File: docs/user-guides/installing-perses-in-a-container.md [hunk 2]
```diff
@@ -28,39 +27,36 @@ $ docker run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
...
-```
+- images built from the main branch:  `persesdev/perses:main-2023-12-14-f66e10ce-distroless-debug`
+  or `persesdev/perses:main-2023-12-14-f66e10ce-distroless`
+- latest Perses image: `persesdev/perses` or `persesdev/perses:latest-debug`
+- precise release image: `persesdev/perses:v0.42.1`, `persesdev/perses:v0.42.1-debug`, `persesdev/perses:v0.42.1-distroless`, `persesdev/perses:v0.42.1-distroless-debug`
+- minor release image: `persesdev/perses:v0.42`, `persesdev/perses:v0.42-debug`, `persesdev/perses:v0.42-distroless`, `persesdev/perses:v0.42-distroless-debug`
 
 ## Connect a browser (default)
 
 Open the Perses console at http://localhost:8080.
```
Comment (Gladorme) [inline docs/user-guides/installing-perses-in-a-container.md:55]:
```suggestion
Open the Perses page at http://localhost:8080.
```
no?

**Review by Gladorme — APPROVED** [review]

File: docs/user-guides/installing-perses-in-a-container.md (hunk 2 above)
Comment (AntoineThebaud) [reply docs/user-guides/installing-perses-in-a-container.md:55]: E.g Openshift also comes with the term console for its UI, but yes I also think that another term like page/website/UI would sound better here.

File: docs/user-guides/installing-perses-in-a-container.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
 # Installing Perses in a container
 
-This sections continues with installing using a container image with Podman open source container tooling, configure
+This section continues with installing using a container image with Podman open-source container tooling, configure
```
Comment (AntoineThebaud) [inline docs/user-guides/installing-perses-in-a-container.md:3]:
```suggestion
This section continues with installing using a container image with Podman open-source container tooling, configuring
```

File: docs/user-guides/installing-perses-in-a-container.md [hunk 4]
```diff
@@ -1,17 +1,16 @@
...
 examples, and run the container on your local machine.
```
Comment (AntoineThebaud) [inline docs/user-guides/installing-perses-in-a-container.md:4]:
```suggestion
examples, and running the container on your local machine.
```

File: docs/user-guides/installing-perses-in-a-container.md [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
 
 ## Installing container tooling
 
-Installing Perses using a container image is going to be demonstrated here using the
-[open source project Podman](https://podman.io/) and [Docker](https://www.docker.com/). It's assumed you have already
-installed and running either the Podman or the Docker commandline tooling.
+Installing Perses using a container image is going to be demonstrated here using[Podman](https://podman.io/) and [Docker](https://www.docker.com/). 
+It's assumed you have already installed and running either the Podman or the Docker commandline tooling.
```
Comment (AntoineThebaud) [inline docs/user-guides/installing-perses-in-a-container.md:9]:
```suggestion
It's assumed you have already installed & are already familiar with either the Podman or the Docker commandline tooling.
```

File: docs/user-guides/installing-perses-in-a-container.md [hunk 6]
```diff
@@ -1,17 +1,16 @@
...
 
 ## Running container image
 
-It's pretty straight forward to running Perses in a container, just start the Perses container image using one of
+It's pretty straight forward to running Perses in a container, start the Perses container image using one of
```
Comment (AntoineThebaud) [inline docs/user-guides/installing-perses-in-a-container.md:13]:
```suggestion
It's pretty straightforward to run Perses in a container: simply start the Perses container image using one of
```

File: docs/user-guides/installing-perses-in-a-container.md [hunk 7]
```diff
@@ -28,39 +27,36 @@ $ docker run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
...
+- minor release image: `persesdev/perses:v0.42`, `persesdev/perses:v0.42-debug`, `persesdev/perses:v0.42-distroless`, `persesdev/perses:v0.42-distroless-debug`
 
 ## Connect a browser (default)
 
 Open the Perses console at http://localhost:8080.
 
-You are presented with an empty home dashboard, in light mode. For fun you can choose optionally to flip the switch in
-the top right corner to enable dark mode.
+You are presented with an empty home dashboard, in light mode.
+For fun, you can choose optionally to flip the switch in the top right corner to enable dark mode.
```
Comment (AntoineThebaud) [inline docs/user-guides/installing-perses-in-a-container.md:58]:
```suggestion
For fun, you can optionally flip the switch in the top right corner to enable dark mode.
```

File: docs/user-guides/installing-perses-in-a-container.md [hunk 8]
```diff
@@ -1,25 +1,20 @@
 # Installing Perses in a container
 
-This sections continues with installing using a container image with Podman open source container tooling, configure
-examples, and run the container on your local machine.
+This section is explaining how you can install Perses using a container image like [Podman](https://podman.io/) or [Docker](https://www.docker.com/).
```
Comment (celian-garcia) [inline docs/user-guides/installing-perses-in-a-container.md:3]: I'd say "using a container manager like"

**Review by celian-garcia — APPROVED** [review]

**Pull request merged**